META.coq.in

```diff
@@ -456,6 +497,9 @@ package "plugins" (
 
     archive(byte)    = "string_notation_plugin.cmo"
     archive(native)  = "string_notation_plugin.cmx"
+
+    plugin(byte)    = "string_syntax_plugin.cmo"
+    plugin(native)  = "string_syntax_plugin.cmxs"
```

Comment:
What is `string_syntax_plugin`?

Reply:
Indeed, that was a typo on my script, fixed. Thanks!